ui/packages/catalog-ui-search/src/main/webapp/react-component/container/user/index.tsx

```diff
@@ -10,6 +9,4 @@
  * <http://www.gnu.org/licenses/lgpl.html>.
  *
  **/
- --}}
-<div class="user-info-region">
-</div>
+export { default } from './user'
```

Comment:
Really odd how github chooses to link old and new files together when all they share is a copyright comment.